lib/graphql/backtrace/table.rb

```diff
@@ -36,6 +36,13 @@ def to_backtrace
 
       private
 
+      def parameter_filter(params)
+        return params if Rails.application.config.filter_parameters.blank?
```

Comment:
graphql-ruby doesn't have a dependence on Rails so `Rails` and `ActionDispatch` can't be referenced like this

Reply:
Ah. I see
So you mean, need to check exists Rails or not?
Hm... I guess add some configuration is better. but `graphql-ruby` doesn't have a configuration. isn't it?

Which way is better?

Reply:
It's not going to be easy for the reason you mentioned... You'd also want this behaviour to be optional as well since not everyone would want it filtered by default.

Reply:
Yes. right. so i suggest "put to configuration". then user can set filters with env.  but unfortunately, don't have a configuration